src/main/dialogs/quickSetStrip/qssTooltipDialog.js

```diff
@@ -57,6 +63,16 @@ fluid.defaults("gpii.app.qssTooltipDialog", {
                     "{that}.model.setting"
                 ]
             }
+        },
+        arrowDirection: {
+            target: "arrowDirection",
+            singleTransform: {
```

Comment:
Very odd - why is there any transform here given that the effect is to return the supplied value? Seems like this is a noop

Reply:
Because we do need to go through the modelRelay, but didn't know how to make it work without the singleTransform part :-). Any guidance?

Reply:
Still very unclear - why does this transform still exist?